wp-postratings.php

```diff
@@ -99,7 +99,7 @@ function the_ratings($start_tag = 'div', $custom_id = 0, $display = true) {
 	if ( (int) $postratings_ajax_style['loading'] === 1 ) {
 		$loading_alt = apply_filters('wp_postratings_loading_alt', '');
 		$loading_alt_html = ! empty( $loading_alt ) ? ' alt="' . esc_attr( $loading_alt ) . '"' : '';
```

Comment:
This is the code part that takes care of the alt tag.
The way you did it will lead in alt tag duplication if `$loading_alt` is set.

Reply:
Fixed, thanks!